src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -55,9 +55,12 @@
 
 TEST_RESOURCE_PURGE_TIMEOUT = timedelta(hours=1)
 TEST_NIGHTLY_CI_RESOURCES_PURGE_TIMEOUT = timedelta(hours=3)  # Buffer for debugging nightly integration test runs
+# See https://docs.databricks.com/en/jobs/parameter-value-references.html#supported-value-references
 EXTRA_TASK_PARAMS = {
     "job_id": "{{job_id}}",
+    "job_name": "{{job.name}}",
```

Comment:
the name should be ucx name, e.g. migration-progress, assessment - not the job name "[UCX][prod] assessment"